src/shape/polyline.rs

```diff
@@ -218,6 +218,7 @@ impl Polyline {
     ///   `num_indices == self.indices.len()`.
     /// - This polyline is oriented counter-clockwise.
     /// - In 3D, the polyline is assumed to be fully coplanar, on a plane with normal given by
```

Comment:
In another commit, someone should fix this comment to complete the sentence to say what that normal is given by.

Reply:
https://github.com/dimforge/parry/pull/247